### ISSUE TITLE
Process hangs when Docker container exits during construction of BmiClientDocker

### ISSUE THREAD

**Issue by sverhoeven** [eWaterCycle/grpc4bmi#50]
When doing:

```
model = BmiClientDocker(image='ewatercycle/wflow-grpc4bmi:latest', image_port=55555,
                        input_dir="./nonexistingpath")
```
The Docker containers dies as wflow is unable to write a log file to the ./nonexisistingpath.

I expected an exception to be thrown that the Docker container dies unexpectly.
What happens is that the BmiClientDocker keeps waiting for the port become available.

By adding `remove=False` as argument and looking with `docker ps -a` and `docker logs <died container id>` I was able determine that wflow was unable to write the log file.

**Comment by Peter9192** [comment]
I seem to get the same problem with an input path that *does* exist: 
`ERROR: Failed to initialize logger with logfile: wflow_bmi.log`

**Comment by Peter9192** [comment]
OK, my problem is solved. I had created the input directory from within a jupyter notebook, and apparently that messed up the permissions.

**Comment by Peter9192** [comment]
Fixed in #53
